src/components/Authentication/Login.jsx

```diff
@@ -0,0 +1,126 @@
+      <div
+        style={{
+          margin: 'auto',
+          textAlign: 'center',
+          width: '598px',
+          minWidth: '300px',
+          padding: '50px, 40px, 40px, 37px',
+          gap: '25px',
+        }}
+      >
+        <h2>Welcome. Please enter login information.</h2>
+        {errorMessage && <p>{errorMessage}</p>}
+        <form onSubmit={handleStdLogin}>
+          <FormControl>
+            <div>
+              <Input
+                style={{ width: '350px', height: '81px', margin: '20px' }}
+                type="email"
+                onChange={({ target }) => setEmail(target.value)}
+                placeholder="Email"
+              />
+              <Input
+                style={{ width: '350px', height: '81px', margin: '20px' }}
+                type="password"
+                onChange={({ target }) => setPassword(target.value)}
+                placeholder="Password"
+              />
+            </div>
+
+            <div
+              style={{
+                marginTop: '25px',
+                marginBottom: '25px',
+              }}
+            >
+              <Button
+                type="submit"
+                style={{
+                  borderRadius: '30px',
+                  marginRight: '24px',
+                  width: '130px',
+                  height: '38px',
+                }}
+                backgroundColor={'#3182CE'}
+                color={'white'}
+              >
+                Login
+              </Button>
+              <Button
+                style={{
+                  borderRadius: '30px',
+                  marginLeft: '24px',
+                  width: '130px',
+                  height: '38px',
+                }}
+                backgroundColor={'#A0AEC0'}
+                color={'white'}
+              >
+                Create Account
+              </Button>
+            </div>
+
+            <div>
+              <a href="/forgotpassword" style={{ fontWeight: 'bold' }}>
+                Forgot Password
+              </a>
+            </div>
+          </FormControl>
+        </form>
+
+        {/* <h2>Welcome. Please enter login information.</h2>
+      {errorMessage && <p>{errorMessage}</p>} */}
+        {/* <form onSubmit={handleStdLogin}>
+        <input type="text" onChange={({ target }) => setEmail(target.value)} placeholder="Email" />
+        <br />
+        <input
+          type="password"
+          onChange={({ target }) => setPassword(target.value)}
+          placeholder="Password"
+        />
+        <br />
+        <a href="/forgotpassword">Forgot Password</a>
+        <br />
+        <button type="submit">Login</button>
+      </form> */}
+
+        {/* <br />
+      <form onSubmit={handleGoogleLogin}>
+        <button type="submit">Sign In with Google</button>
+      </form>
+      <br /> */}
+      </div>
+    </Center>
+  );
+};
+
+Login.propTypes = {
+  cookies: instanceOf(Cookies).isRequired,
+};
+
+export default withCookies(Login);
```

Comment:
⚠️ **[eslint]** <react-refresh/only-export-components> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Fast refresh can't handle anonymous components. Add a name to your export.